package.json

```diff
@@ -87,7 +87,7 @@
     "nib": "^1.1.0",
     "node-dir": "0.1.16",
     "npm": "^5.3.0",
-    "panoptes-client": "^2.2.0",
+    "panoptes-client": "^2.9.0",
```

Comment:
Should this be "^2.9" so that patches to 2.9 are installed automatically?

Reply:
I'm not sure. PFE doesn't do that.